core/mutator.ts

```diff
@@ -265,7 +265,7 @@ export class Mutator extends Icon {
     const canvas = this.workspace_.getCanvas();
     const workspaceSize = canvas.getBBox();
     let width = workspaceSize.width + workspaceSize.x;
-    let height = workspaceSize.height + doubleBorderWidth * 3;
+    let height = workspaceSize.height + doubleBorderWidth * 2;
```

Comment:
Why this seemingly unrelated change to `Mutator`?

Reply:
Accidental commit, thanks for the catch!